### PR TITLE
implementing tflxmertmodel integration test

### PR DESCRIPTION
**Pull request by sadakmed**
What does this PR do?
this PR implement an integration test for TFLxmertmodel as requested in [#9954](https://github.com/huggingface/transformers/issues/9954)

@LysandreJik

### CODE REVIEW

**Comment by LysandreJik** [comment]
I believe your version of the code quality tools isn't up to date - can you try running the following at the root of your clone: `pip install -e .[quality] -U` to update your tools?

**Comment by sadakmed** [comment]
Hi @LysandreJik  Yep It was my black version

File: tests/test_modeling_tf_lxmert.py [hunk 1]
```diff
@@ -751,20 +875,68 @@ def test_saved_model_creation_extended(self):
...
+        expected_slice = tf.constant(
+            [
+                [
+                    [0.24170142, -0.98075, 0.14797261],
+                    [1.2540525, -0.83198136, 0.5112344],
+                    [1.4070463, -1.1051831, 0.6990401],
+                ]
+            ]
+        )
+        tf.debugging.asset_near(output[:, :3, :3], expected_slice, atol=1e-4)
```
Comment (LysandreJik) [inline tests/test_modeling_tf_lxmert.py:942]: This should be `assert_near`
```suggestion
        tf.debugging.assert_near(output[:, :3, :3], expected_slice, atol=1e-4)
```

File: tests/test_modeling_tf_lxmert.py [hunk 2]
```diff
@@ -751,20 +875,68 @@ def test_saved_model_creation_extended(self):
...
                         self.model_tester.num_attention_heads,
                         self.model_tester.num_visual_features,
                         self.model_tester.num_visual_features,
                     ],
-                    [self.model_tester.num_attention_heads, encoder_key_length, self.model_tester.num_visual_features],
+                    [
+                        self.model_tester.num_attention_heads,
+                        encoder_key_length,
+                        self.model_tester.num_visual_features,
+                    ],
```
Comment (LysandreJik) [inline tests/test_modeling_tf_lxmert.py:911]: There's a lot of style changes in your PR - do you think you could only do the changes relative to the code you're adding? Thank you!

**Review by LysandreJik — APPROVED** [review]
Great, thanks @sadakmed!

**Pull request merged**